app/views/legislation/proposals/_flag_actions.html.erb

```diff
@@ -5,7 +5,7 @@
         <span class="icon-flag flag-disable"></span>
       </a>
       <span class="dropdown-pane" id="flag-drop-proposal-<%= proposal.id %>" data-dropdown data-auto-focus="true">
-        <%= link_to t("shared.flag"), flag_proposal_path(proposal), method: :put, remote: true, id: "flag-proposal-#{proposal.id}" %>
+        <%= link_to t("shared.flag"), flag_legislation_process_proposal_path(proposal), method: :put, remote: true, id: "flag-proposal-#{proposal.id}" %>
```

Comment:
A parameter is missing here: it should be `flag_legislation_process_proposal_path(proposal.process, poposal)`